charts/pulsar/templates/broker-statefulset.yaml

```diff
@@ -49,7 +49,7 @@ spec:
   podManagementPolicy: {{ $stsObj.spec.podManagementPolicy }}
   {{- else }}
   {{- if not .Values.components.functions }}
-  podManagementPolicy: Parallel
+  podManagementPolicy: {{ .Values.broker.podManagementPolicy }}
   {{- else }}
   podManagementPolicy: OrderedReady
   {{- end }}
```

Comment:
I think that we'd need to take this existing logic into account. The default value in the values.yaml should be absent (nil). When the value is absent, the former logic should be used. If it's set, it would get applied.